dockers/docker-snmp-sv2/config.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+snmp_user=snmp
+
+#TODO: check with sasha whether this file will be generated by base image or in container
```

Comment:
>this file [](start = 32, length = 9)

Do you mean var.yml?
The name is meaningless to me. Would you also change sonic-mgmt before this PR?